cmd/crictl/container_test.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"k8s.io/kubernetes/pkg/kubelet/types"
+	"testing"
+	"time"
+
+	"github.com/go-test/deep"
+	"github.com/stretchr/testify/assert"
```

Comment:
I see it is used in k/k in places. But I am not sure how badly this repo needs another dependency

Reply:
ok I'll make it use ginkgo